src/class-asset-manager.php

```diff
@@ -245,7 +229,7 @@ public function set_asset_type_defaults() {}
 	public function add_hooks() {
 		foreach ( $this->load_hooks as $hook => $functions ) {
 			foreach ( $functions as $function => $priority ) {
-				add_action( $hook, [ $this, $function ], $priority );
+				add_action( $hook, [ $this, $function ], $priority ); // @phpstan-ignore-line expects callable
```

Comment:
`assert` is a noop in production.
```suggestion
				assert( method_exists( $function, $this ) );
				add_action( $hook, [ $this, $function ], $priority );
```

Reply:
Annotations are much better for automatic hooking.
https://github.com/szepeviktor/SentencePress/blob/master/src/HookAnnotation.php
... or modern PHP attributes
https://github.com/boxuk/wp-hook-attributes?tab=readme-ov-file#usage

Reply:
I usually do it with https://mantle.alley.com/docs/features/support/hookable#usage-with-attributes but this is an older plugin

Reply:
📡 What a space laser you have there!